frontend/micro-ui/web/micro-ui-internals/packages/modules/pt/src/pages/citizen/Create/AssessmentDetailsSection.js

```
@@ -0,0 +1,88 @@
+      <div style={{ ...styles.poppinsLabel, color: "#888" }}>{t("Plot area")}</div>
+      <TextInput
+        style={styles.widthInput}
+        name="plotArea"
+        value={assessmentDetails.plotArea}
+        onChange={handleAssessmentInputChange}
+        placeholder={t("Enter")}
+        disable={true}
+      />
+    </div>
+
+    <div style={styles.flex30}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)